### ISSUE TITLE
Feature mapping loss increasing during training

### ISSUE THREAD

**Issue by Ziyi6** [jik876/hifi-gan#175]
Hello, I run the training script with config v1 and LJ-Speech dataset and I plotted all the loss separately during training. I found out that the feature mapping loss of mpd and msd kept increasing during training. Is it normal? How can one understand such a phenomenon? Should the loss decreasing during training?

<img width="1759" height="927" alt="Image" src="https://github.com/user-attachments/assets/0e4789fb-5416-4134-9813-3ea199686742" />

**Comment by TheDevloper2023** [comment]
It should lower after a few thousand steps
